Assets/MRTK/Core/Services/BaseBoundarySystem.cs

```diff
@@ -61,6 +61,8 @@ private void ReadProfile()
         /// <inheritdoc/>
         public override void Initialize()
         {
+            base.Initialize();
```

Comment:
Inconsistent with the other `base.Initialize()` changes. This one is at the beginning of the method, while the others have mostly been moved to the end.

Reply:
missed that one. i would love to hear your thoughts on the earlier comment.